Fix language packs with broken summary translation instance.

EnTeQuAk opened the pull request:
Fixes #5432 

What happened on our production system:

The `zadmin.tasks.fetch_langpack` task set `summary` to the
same translation instance of `name` for add-ons without a
summary set.
This meant that changing `name` or `summary` automatically
changed the other one.

eviljeff commented on src/olympia/zadmin/tasks.py:
```diff
@@ -615,8 +615,9 @@ def fetch_langpack(url, xpi, **kw):
-                     summary=(addon.summary if addon.summary else addon.name))
+        addon.status = amo.STATUS_PUBLIC
+        addon.summary = addon.summary if addon.summary else unicode(addon.name)
+        addon.save(update_fields=('status', 'summary'))
```
I'm scratching my head why this does something different than the `.update` statement but I believe you

EnTeQuAk replied:
to be honest, I did so too, I ran into some very weird unicode errors along the way which I'm still debugging but I didn't want to block myself on them.

EnTeQuAk commented:
> Would it be lots of work to to have the cron job run on -dev before you merge so you can run the command and check it fixes them?

Good plan, I'll do that shortly.

EnTeQuAk commented:
```
>>> UserProfile.objects.get(email=settings.LANGPACK_OWNER_EMAIL).addons.filter(Q(summary_id=F('name_id'))).count()
63
```

Just ran the tasks on -dev, some might still run but I'm going to merge this and test it's working on -dev afterwards.

EnTeQuAk commented:
Before:

```
>>> UserProfile.objects.get(email=settings.LANGPACK_OWNER_EMAIL).addons.filter(Q(summary_id=F('name_id'))).count()
64
```

Running:

```
Attempt to fix 417146: Acholi (UG) Language Pack
fixed 417146: Acholi (UG) Language Pack
Attempt to fix 417148: Assamese (IN) Language Pack
fixed 417148: Assamese (IN) Language Pack
Attempt to fix 417150: Asturianu Language Pack
fixed 417150: Asturianu Language Pack
Attempt to fix 417154: български Language Pack
fixed 417154: български Language Pack
Attempt to fix 417156: Bengali (India) Language Pack
fixed 417156: Bengali (India) Language Pack
Attempt to fix 417158: Bosanski Language Pack
fixed 417158: Bosanski Language Pack
Attempt to fix 417160: Czech (CZ) Language Pack
fixed 417160: Czech (CZ) Language Pack
Attempt to fix 417162: Cymraeg Language Pack
fixed 417162: Cymraeg Language Pack
Attempt to fix 417164: Deutsch (DE) Language Pack
fixed 417164: Deutsch (DE) Language Pack
Attempt to fix 417166: Greek (GR) Language Pack
fixed 417166: Greek (GR) Language Pack
Attempt to fix 417168: English (GB) Language Pack
fixed 417168: English (GB) Language Pack
Attempt to fix 417170: Español (AR) Language Pack
fixed 417170: Español (AR) Language Pack
Attempt to fix 417172: Español (España) Language Pack
fixed 417172: Español (España) Language Pack
Attempt to fix 417174: Basque Language Pack
fixed 417174: Basque Language Pack
Attempt to fix 417176: Finnish Language Pack
fixed 417176: Finnish Language Pack
Attempt to fix 417178: Français Language Pack
fixed 417178: Français Language Pack
Attempt to fix 417180: Frysk (fy) Language Pack
fixed 417180: Frysk (fy) Language Pack
Attempt to fix 417182: Irish (IE) Language Pack
fixed 417182: Irish (IE) Language Pack
Attempt to fix 417184: Gujarati (IN) Language Pack
fixed 417184: Gujarati (IN) Language Pack
Attempt to fix 417186: Hebrew (IL) Language Pack
fixed 417186: Hebrew (IL) Language Pack
Attempt to fix 417188: Hindi Language Pack
fixed 417188: Hindi Language Pack
Attempt to fix 417190: Hrvatski (HR) Language Pack
fixed 417190: Hrvatski (HR) Language Pack
Attempt to fix 417192: Icelandic (IS) Language Pack
fixed 417192: Icelandic (IS) Language Pack
Attempt to fix 417194: Italiano (IT) Language Pack
fixed 417194: Italiano (IT) Language Pack
Attempt to fix 417196: Japanese Language Pack
fixed 417196: Japanese Language Pack
Attempt to fix 417198: Kazakh (KZ) Language Pack
fixed 417198: Kazakh (KZ) Language Pack
Attempt to fix 417200: ខ្មែរ Language Pack
fixed 417200: ខ្មែរ Language Pack
Attempt to fix 417202: Kannada (IN) Language Pack
fixed 417202: Kannada (IN) Language Pack
Attempt to fix 417204: Korean (KR) Language Pack
fixed 417204: Korean (KR) Language Pack
Attempt to fix 417210: Lietuvių Language Pack
fixed 417210: Lietuvių Language Pack
Attempt to fix 417212: Maithili Language Pack
fixed 417212: Maithili Language Pack
Attempt to fix 417214: Македонски Language Pack
fixed 417214: Македонски Language Pack
Attempt to fix 417216: Malayalam (IN) Language Pack
fixed 417216: Malayalam (IN) Language Pack
Attempt to fix 417218: Marathi Language Pack
fixed 417218: Marathi Language Pack
Attempt to fix 417220: Norsk bokmål (NO) Language Pack
fixed 417220: Norsk bokmål (NO) Language Pack
Attempt to fix 417222: Nederlands (NL) Language Pack
fixed 417222: Nederlands (NL) Language Pack
Attempt to fix 417224: Norsk nynorsk (NO) Language Pack
fixed 417224: Norsk nynorsk (NO) Language Pack
Attempt to fix 417226: Oriya (OR) Language Pack
fixed 417226: Oriya (OR) Language Pack
Attempt to fix 417228: Punjabi (Gurmukhi) Language Pack
fixed 417228: Punjabi (Gurmukhi) Language Pack
Attempt to fix 417230: Polski Language Pack
fixed 417230: Polski Language Pack
Attempt to fix 417232: Português Portugal Language Pack
fixed 417232: Português Portugal Language Pack
Attempt to fix 417234: Rumantsch Language Pack
fixed 417234: Rumantsch Language Pack
Attempt to fix 417236: Russian (RU) Language Pack
fixed 417236: Russian (RU) Language Pack
Attempt to fix 417238: Slovak (SK) Language Pack
fixed 417238: Slovak (SK) Language Pack
Attempt to fix 417240: Slovenski jezik Language Pack
fixed 417240: Slovenski jezik Language Pack
Attempt to fix 417242: Soŋay Language Pack
fixed 417242: Soŋay Language Pack
Attempt to fix 417244: Shqip Language Pack
fixed 417244: Shqip Language Pack
Attempt to fix 417246: српски (sr) Language Pack
fixed 417246: српски (sr) Language Pack
Attempt to fix 417248: Svenska (SE) Language Pack
fixed 417248: Svenska (SE) Language Pack
Attempt to fix 417250: Tamil Language Pack
fixed 417250: Tamil Language Pack
Attempt to fix 417252: Thai Language Pack
fixed 417252: Thai Language Pack
Attempt to fix 417254: Türkçe (TR) Language Pack
fixed 417254: Türkçe (TR) Language Pack
Attempt to fix 417256: Chinese Simplified (zh-CN) Language Pack
fixed 417256: Chinese Simplified (zh-CN) Language Pack
Attempt to fix 417258: Traditional Chinese (zh-TW) Language Pack
fixed 417258: Traditional Chinese (zh-TW) Language Pack
Attempt to fix 494458: Azərbaycanca (AZ) Language Pack
fixed 494458: Azərbaycanca (AZ) Language Pack
Attempt to fix 494459: Kaqchikel (Guatemala) Language Pack
fixed 494459: Kaqchikel (Guatemala) Language Pack
Attempt to fix 494460: Dolnoserbšćina Language Pack
fixed 494460: Dolnoserbšćina Language Pack
Attempt to fix 494461: Guarani (Paraguay) Language Pack
fixed 494461: Guarani (Paraguay) Language Pack
Attempt to fix 494462: Hornjoserbšćina Language Pack
fixed 494462: Hornjoserbšćina Language Pack
Attempt to fix 494463: ინგლისური (აშშ) Language Pack
fixed 494463: ინგლისური (აშშ) Language Pack
Attempt to fix 494464: Bahasa Melayu (ms) Language Pack
fixed 494464: Bahasa Melayu (ms) Language Pack
Attempt to fix 494465: Taqbaylit Language Pack
fixed 494465: Taqbaylit Language Pack
Attempt to fix 494466: Uzbek Language Pack
fixed 494466: Uzbek Language Pack
Attempt to fix 494467: اردو (پاکستان) Language Pack
fixed 494467: اردو (پاکستان) Language Pack
```

After:

```
>>> UserProfile.objects.get(email=settings.LANGPACK_OWNER_EMAIL).addons.filter(Q(summary_id=F('name_id'))).count()
0
```
```
